lexos/processors/prepare/cutter.py

```diff
@@ -407,18 +407,17 @@ def cut_by_number(text, num_chunks):
     return string_list
 
 
-def cut_by_milestone(text, cutting_value):
-    """
-    Cuts the file into as many chunks as there are instances of the
-        substring cuttingValue.  Chunk boundaries are made wherever
-        the string appears.
-    Args: text -- the text to be chunked as a single string
+def cut_by_milestone(text: str, cutting_value: str) -> List[str]:
+    """Cuts the file into chunks by milestones and made chunk boundaries
 
-    Returns: A list of strings which are to become the new chunks.
+    Chunk boundaries should be created when every milestone appears
+    :param text: the text to be chunked as a single string
+    :param cutting_value: # maybe we should change this variable name?
+    :return: A list of strings which are to become the new chunks.
     """
     chunk_list = []  # container for chunks
     len_milestone = len(cutting_value)  # length of milestone term
-    cutting_value = cutting_value
+    cutting_value = cutting_value   # maybe we should delete this line?
```

Comment:
use `# TODO: maybe we should delete this line?` so that we can search this in pycharm.
Thanks